concept/thing/attribute.feature

```diff
@@ -308,6 +308,19 @@ Feature: Concept Attribute
 
   Scenario: Attribute with value type double can be owned
 
-  Scenario: Attribute with value type string can be owned
+  Scenario: Attribute with value type datetime can be inserted in one timezone and queried in another timezone with no change in the value
+    When set time-zone is: Asia/Calcutta
+    When $x = attribute(birth-date) as(datetime) put: 2001-08-23 08:30:00
+    Then attribute $x is null: false
+    Then attribute $x has type: birth-date
+    Then attribute $x has value type: datetime
+    Then attribute $x has datetime value: 2001-08-23 08:30:00
+    When transaction commits
+    When session opens transaction of type: read
+    When set time-zone is: America/Chicago
+    When $x = attribute(birth-date) as(datetime) get: 2001-08-23 08:30:00
+    Then attribute $x is null: false
+    Then attribute $x has type: birth-date
+    Then attribute $x has value type: datetime
+    Then attribute $x has datetime value: 2001-08-23 08:30:00
 
-  Scenario: Attribute with value type datetime can be owned
```

Comment:
Did you mean to delete the two scenarios?  Please restore.

Reply:
But there was no test corresponding to this scenario

Reply:
Yes, but I see other tests which you've not removed.
So we should either remove all of them or leave all of them.